### DISCUSSION TITLE
Rrule Prisma Model

### DISCUSSION THREAD

**Discussion by laurent512** [prisma/prisma#16986] [Q&A]
Hello, 

I am trying to manage reccurent events with exception.
Some research led me to adopting RRule conventions . 

Would anyone have a compliant database design model available for me ? :)
I would like to avoid making it myself with some mistakes :/

Thank you so much

**Comment by laurent512** [comment]
Based on http://jakubroztocil.github.io/rrule/ I made this : 

```
model Rrule {
  id         Int        @id @default(autoincrement())
  freq       FREQ_RRULE
  dtstart    DateTime
  tzid       Int
  until      DateTime?
  interval   Int
  wkst       Int[]
  byweekday  Int[]
  bymonth    Int[]
  bysetpos   Int[]
  bymonthday Int[]
  byyearday  Int[]
  byweekno   Int[]
  byhour     Int[]
  byminute   Int[]
  bysecond   Int[]
}

enum FREQ_RRULE {
  YEARLY
  MONTHLY
  WEEKLY
  DAILY
  HOURLY
  MINUTELY
  SECONDLY
}

```

Hope this is fine , if anyone knows how to proceed for exception I would appreciate it ;)

**Reply by pantharshit00** [reply]
The model looks fine to me for this use case.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
